@ecomplus/widget-ebit/cms.config.js

```diff
@@ -0,0 +1,47 @@
+import { name } from './package.json'
+
+export default () => ({
+  label: 'Chat Tawk.to',
+  fields: [
+    {
+      name: 'pkg',
+      widget: 'hidden',
+      default: name
+    },
+    {
+      label: 'Ativar',
+      hint: 'Instalar Ebit',
+      name: 'active',
+      widget: 'boolean'
+    },
+    {
+      name: 'desktopOnly',
+      widget: 'hidden',
+      default: false
+    },
+    {
+      name: 'enableCheckout',
+      widget: 'hidden',
+      default: true
+    },
+    {
+      label: 'Opções',
+      name: 'options',
+      widget: 'object',
+      hint: 'Configurar ebit',
```

Comment:
```suggestion
      hint: 'Personalizações do widget',
```

Possivelmente a gente coloca outras coisas no objeto `options`, não necessariamente relacionadas ao Ebit, sacou? Uma customização na posição onde ele aparece, por exemplo...

Reply:
entendi